test/test_runner/ecs_daemon_base_test_runner.go

```diff
@@ -0,0 +1,122 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Same here.. this looks identical to our existing runner so I would just reuse.

Reply:
trimmed down so that there are only what's needed